src/components/alert/useAlertMessage.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react'
+
+export const useAlertMessage = () => {
+
+   const [alertMessage, setAlertMessage] = useState('')
+   const [backgroundColor, setBackgroundColor] = useState('')
+
+   const handleAlertMessage = (alertMessage, type) => {
+
+      const dictionary = {
```

Comment:
This constant is going to be created on each re-render, move the const outside of the custom hook.

Reply:
we can do this because the constant values are static and not dynamic